x-pack/plugins/beats/server/lib/adapters/beats/elasticsearch_beats_adapter.ts

```diff
@@ -98,7 +82,10 @@ export class ElasticsearchBeatsAdapter implements CMBeatsAdapter {
       type: '_doc',
     };
     const response = await this.framework.callWithRequest(req, 'mget', params);
-    return get(response, 'docs', []);
+
+    return get(response, 'docs', [])
```

Comment:
That is a fair argument. Any thoughts on a better name? I just used get because it is a CRUD term, and it did not duplicate as I thought a beats adapter, with all of its methods referring to beats might feel odd to say getBeats... So get was used for lack of imagination on my part.

Reply:
Well, I think `get` is the most appropriate name for the adapter function and don't think it should change. We could alias the `lodash` function. Like I said, I don't think it's a huge deal because anyone who looks at these `get` calls will figure out pretty quickly what is going on, but if we made the change, calling it `_get` or something that delineates it from the adapter function would eliminate the potential confusion.

Reply:
ah good idea